include/dts-functions.sh

```diff
@@ -1442,7 +1460,9 @@ main_menu_options(){
       [ "${SYSTEM_VENDOR}" = "QEMU" ] || [ "${SYSTEM_VENDOR}" = "Emulation" ] && return 0
 
       if check_if_dasharo; then
-        ${CMD_DASHARO_DEPLOY} restore
+        if ! ${CMD_DASHARO_DEPLOY} restore; then
+          send_dts_logs ask
```

Comment:
```suggestion
          send_dts_logs ask return 0
```

So not to see `Press Enter to continue.` twice.

Reply:
d394c656fe39